sweeper.lua

```diff
@@ -0,0 +1,45 @@
+local S = moreblocks.intllib
+local check_protection = moreblocks.check_protection
+local sweepables = {}
+
+-- example sweepable node
+-- TODO API for sweepable node registration
+sweepables["default:mossycobble"] = function(pos, node)
```

Comment:
A shared function for all sweepables would be preferable to avoid having lua reallocate and recompile the same function each time you register a new node.

You could have the tool check for `on_sweep` in the node definition.
If a function is present, call it, if a string with a node name is there, use that as replacement target.
This would also allow nodes to define custom sweep behavior without dependency on moreblocks.

Reply:
Alternatively just have a KV table map from sweepables to replace nodes.